scalableminds/webknossos
# Reload all layers when render missing data black is changed

This PR removes the warning to reload the page when toggling "render missing data black". It just automatically reloads all layers instead. 

### URL of deployed dev instance (used for testing):
- https://reloadlayeronrendermissingdatablack.webknossos.xyz

### Steps to test:
- Open a tracing and toggle the option "render missing data black".
- All layers should be fetched from the backend again and no reloading of the page is necessary for the changes to take effect.

### Issues:
- fixes #4454

------
